game/view/gameplay/actionhud/controlhint.lua

```diff
@@ -0,0 +1,70 @@
+
+local FONT    = require 'view.helpers.font'
+local RES     = require 'resources'
+local PROFILE = require 'infra.profile'
+local COLORS  = require 'domain.definitions.colors'
+
+local Class   = require "steaming.extra_libs.hump.class"
+local ELEMENT = require "steaming.classes.primitives.element"
+local vec2    = require 'cpml' .vec2
+
+local ControlHint = Class{
+  __includes = { ELEMENT }
+}
+
+function ControlHint:init(x, y, texture_name, hints)
+    ELEMENT.init(self)
+    self:setSubtype("control_hints")
+
+    self.pos = vec2(x, y)
+    self.hints = hints
+    self.mode = "DEFAULT"
+
+    self.show = PROFILE.getTutorial("finished_tutorial")
```

Comment:
i highly recommend to not use magical literals. always prefer to store them somewhere

Reply:
Also too big for this PR, since many other parts of the code rely on `PROFILE.getTutorial()`. Issue #1459 will address this particular case of magical literals.